Policies/github-terms-of-service.md

```diff
@@ -36,95 +36,118 @@ Thank you for using GitHub! We're happy you're here. Please read this Terms of S
 
 #### 2. Required Information
-You must provide a valid email address in order to complete the signup process. Any other information requested, such as your real name, is optional, unless you are accepting these terms on behalf of a legal entity (in which case we need more information about the legal entity) or if you opt for a [paid Account](#k-payment), in which case additional information will be necessary for billing purposes.
+The following information is required:
+- A valid email address to complete the signup process. 
+- If you represent a legal entity, certain information about the legal entity.
+- For [paid Accounts](#k-payment), certain information for billing purposes.
+
+Other information requested (which could include your real name) is optional.
 
 #### 3. Account Requirements
-We have a few simple rules for User Accounts on GitHub's Service.
-- You must be a human to create an Account. Accounts registered by "bots" or other automated methods are not permitted. We do permit machine accounts:  
-- A machine account is an Account set up by an individual human who accepts the Terms on behalf of the Account, provides a valid email address, and is responsible for its actions. A machine account is used exclusively for performing automated tasks. Multiple users may direct the actions of a machine account, but the owner of the Account is ultimately responsible for the machine's actions. You may maintain no more than one free machine account in addition to your free User Account.
-- One person or legal entity may maintain no more than one free Account (if you choose to control a machine account as well, that's fine, but it can only be used for running a machine).
-- You must be age 13 or older. While we are thrilled to see brilliant young coders get excited by learning to program, we must comply with United States law. GitHub does not target our Service to children under 13, and we do not permit any Users under 13 on our Service. If we learn of any User under the age of 13, we will [terminate that User’s Account immediately](#l-cancellation-and-termination). If you are a resident of a country outside the United States, your country’s minimum age may be older; in such a case, you are responsible for complying with your country’s laws.
+- You must be a human to create an Account. Accounts registered by "bots" or other automated methods are not permitted. We do permit machine accounts. When you need to deal with, or know about, machine accounts, make sure you have read the following separately bullet-pointed text (otherwise you can skip it):  
```

Comment:
You can use `details` HTML tag for a collapsable block.

Reply:
That sounds like a great idea.. 👍  Perhaps someone else can improve the PR such that the `details` HTML tag is used for this and possibly also in other relevant places too?

Reply:
I've now updated the patch so that it includes this improvement. Not sure what the relationship now is with this PR. Anyone know whether I have to now make a second PR?